README.md

```diff
@@ -50,15 +50,14 @@ These are args currently supported:
     kubectl rook-ceph --context=$(kubectl config current-context) mons
     ```
 
-
 ### Commands
 
 - `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.
 
 - `rbd <args>` : Call a 'rbd' CLI command with arbitrary args
 
 - `mons` : Print mon endpoints
-  - `restore-quorum <mon-name>` : Restore the mon quorum based on a single healthy mon since quorum was lost with the other mons
```

Comment:
I imagine there are scenarios where the `ceph mon stat` would fail, but the user still wants to reset the quorum. I'd suggest instead of removing the mon name param, let's make it optional. Then we run the `ceph mon stat` and if it fails or doesn't match the mon (if they specify it), then we ask if they really want to continue.

Reply:
I see your point, I have made it optional. Thanks